daemon/cmd/daemon_main.go

```diff
@@ -279,10 +279,6 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.String(option.IPv6MCastDevice, "", "Device that joins a Solicited-Node multicast group for IPv6")
 	option.BindEnv(vp, option.IPv6MCastDevice)
 
-	flags.Bool(option.EnableRemoteNodeIdentity, defaults.EnableRemoteNodeIdentity, "Enable use of remote node identity")
```

Comment:
Please update the `cmdref` docs (by running `make -C Documentation update-cmdref`) in the same commit which removes the flag. Thanks!

Reply:
This is not resolved.